packages/shared/src/components/sidebar/Sidebar.tsx

```diff
@@ -74,6 +110,18 @@ export default function Sidebar({
     [sidebarExpanded, sidebarRendered, activePage],
   );
 
+  useEffect(() => {
+    if (squadVersion !== SquadVersion.Off && squadVisible) {
+      trackEvent({
```

Comment:
My only concern is that it will be triggered multiple times, but we can leave it as it is for now. I can deal with it

Reply:
Yeah was indeed in two minds about using effect like this, it is unlikely it would change in one session though from my tests.

Reply:
Ok then :)